docs/pages/jpscore/jpscore_inifile.md

```diff
@@ -6,267 +6,1219 @@ sidebar: jupedsim_sidebar
+    <route_choice_models>
+        <!-- Definition of routing strategies -->
+    </route_choice_models>
 
--   `<schedule_file>schedule.xml</schedule_file>`
-    The name and location of the schedule file. Schedules can be used to group doors and open or close this
-    groups of door at certain points of time. See [schedule_file](jpscore_schedule.html)
+    <operational_models>
+        <!-- Interaction of agents and walls -->
+    </operational_models>
 
--   `<show_statistics>true</show_statistics>` Show different aggregate statistics e.g. the usage of the doors. (default: false)
+    <!-- OPTIONAL settings -->
 
--   `<output path="output_directory" />`
-    The name and location where the results of the simulation should be stored. The program creates a folder with the
-    given name and copies all needed files of the simulation to this folder. Additionally the file names in the ini and
-    geometry file are updated. Resulting in a folder with the results and everything to reproduce a specific simulation.
-    If no output path is given the folder is named `results`.
+    <routing>
+        <!-- Parameters to adapt routing via goals-->
+    </routing>
 
--  The trajectory file
+    <traffic_constraints>
+        <!-- Traffic information to adapt the flow -->
+    </traffic_constraints>
 
-```bash
- <trajectories format="plain" fps="8" precision="2" color_mode="velocity">
-    <file location="trajectories.txt" />
- </trajectories>
+    <train_constraints>
+        <!-- Parameters for simulation of train arrival/departure -->
+    </train_constraints>
+</JuPedSim>
 ```
 
-The options for the format are
+# Header
+
+## Required Header Settings
+
+The following elements must be defined in the header:
+
+- `<seed>seed</seed>`
+
+  Set the  `seed` value of the random number generator to `s`. If the tag is empty or missing, the current
+  time (`time(NULL)`) is used i.e. random initial conditions.
+
+- `<max_sim_time>t</max_sim_time>`
+  the maximal simulation time in seconds.
+
+- `<geometry>geometry.xml</geometry>`
+  The name and location of the geometry file. All file locations are relative to the actual location of the project
+  file. See [specification](jpscore_geometry.html) of the geometry format.
+
+- `<output path="output_directory" />`
+  The name and location where the results of the simulation should be stored. The program creates a folder with the
+  given name and copies all needed files of the simulation to this folder. Additionally the file names in the ini and
+  geometry file are updated. Resulting in a folder with the results and everything to reproduce a specific simulation.
+  If no output path is given the folder is named `results`.
+
+- The `trajectory` file with its location to configure output as described below.
+
+### Trajectory settings
 
-- `plain`: simple text format. See section [plain](jpscore_trajectory#txt).
 - `fps`: defines the frame rate per second for the trajectories.
```

Comment:
Some additional explanation should be provided:
The simulation runs at a specified interval <insert ref to parameter>, however not every frame will be written to the trajectory file. Instead trajectories are written out with `fps` many frames per second.

Note: The logic is a major problem we need to address anyways, because we are adjusting the simulation interval to match fps...

Reply:
Done